wiki/markdown/step-by-step-subscription-notification.md

```diff
@@ -167,9 +167,8 @@ value of the testcase id. The complete notification message can be seen below:
 ## Failed notifications
 
 If the notification via REST POST fails, then a fixed number of attempts are
-made to resend successfully. The number of attempts are specified in the
-[application.properties](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/resources/application.properties)
-as “notification.retry”. If message sending attempts fails for the
+made to resend successfully. The number of attempts is specified by a property
+described [here](configuration.md#failed-notifications). If message sending attempts fails for the
```

Comment:
is specified by --> is configured with ? I just thought it read better but it's up to you

Reply:
Done